ui/app/services/notifications.js

```diff
@@ -29,7 +29,7 @@ export default class NotificationsService extends FlashService {
     }
 
     if (!('sticky' in notificationObject)) {
-      notificationObject.sticky = true;
+      notificationObject.sticky = false;
     }
```

Comment:
`false` seems to be the [default for `sticky`](https://github.com/adopted-ember-addons/ember-cli-flash#custom-messages-api) so we can probably remove this altogether?

Reply:
Good call; removed!